### PR TITLE
Cleaning up the CRIB Integration Tests worfklow and bump versions

### PR DESCRIPTION
**Pull request by njegosrailic**
Cleaning up the CRIB Integration Tests workflow. Some of the steps and redundant and the GAP for CRIB isn't being used at all.  

There are two composite GitHub Actions that can be used in your workflow to handle the provisioning and cleanup of ephemeral CRIB environments, so no need to install NIX:

- [crib-deploy-environment action](https://github.com/smartcontractkit/.github/tree/main/actions/crib-deploy-environment) - Composite action for deploying a CRIB, setting up GAP, Nix, and deploying to
  an ephemeral environment.

- [crib-purge-environment](https://github.com/smartcontractkit/.github/tree/main/actions/crib-purge-environment) - Action to destroy CRIB ephemeral environment.
  It requires to run crib-deployment-environment beforehand
  and depends on the environment setup from a dependent composite action.

### CODE REVIEW

File: .github/workflows/crib-integration-test.yml [hunk 1]
```diff
@@ -40,24 +36,6 @@ jobs:
           gc-host: ${{ secrets.GRAFANA_INTERNAL_HOST }}
           gc-org-id: ${{ secrets.GRAFANA_INTERNAL_TENANT_ID }}
 
-      - name: setup-gap k8s
```
Comment (chainchad) [inline .github/workflows/crib-integration-test.yml:43]: I believe this is necessary but @skudasov can confirm.

**Review by chainchad — CHANGES_REQUESTED** [review]

File: .github/workflows/crib-integration-test.yml (hunk 1 above)
Comment (njegosrailic) [reply .github/workflows/crib-integration-test.yml:43]: I wasn't able to find in the code, but yeah he may be pulling some statuses from the API.

Comment (skudasov) [reply .github/workflows/crib-integration-test.yml:43]: It is necessary, how we then connect? We connect using GAP headers through sigv4proxy @njegosrailic

Comment (skudasov) [reply .github/workflows/crib-integration-test.yml:43]: I connect here
```
          GAP_URL: ${{ secrets.GAP_URL }}
```

Comment (njegosrailic) [reply .github/workflows/crib-integration-test.yml:43]: Thanks.

**Review by chainchad — APPROVED** [review]

**Pull request closed without merging**